### ISSUE TITLE
[enhancement] use a db to store data

### ISSUE THREAD

**Issue by grokwithrahul** [renz-b/RoxasCityCOVID19Tweepy#2]
this will make it easier to read/write data and will be more efficient to deal with

if you want, I can use TinyDB to create a JSON-like storage system. I'll write the modules and handler and doc it, so you can access it to write the tweet/store data from tesseract

**Comment by renz-b** [comment]
Hi! Thank you for the efforts that you have put in. Currently, I am busy with a certain personal matter and I will read the code and merge it  at the soonest possible time that I am free. Wow, I did not know TinyDB exists I am familiar and able to create SQLite DB that is built in to python but I thought its too much for such a small project. Haha!

Thank you again

**Comment by grokwithrahul** [comment]
welcome!

tinydb is tiny, and can be used for tiny projects :)

**Comment by grokwithrahul** [comment]
Do you have gitter? It would be easier to discuss on that platform, rather than on issues and pull requests

**Comment by renz-b** [comment]
Nope, I'll create an account if you are more comfortable on that platform. Will come back to you. From my estimate I'll be free in minimum tomorrow and maximum the day after tomorrow.

Sorry for my schedule...

**Comment by grokwithrahul** [comment]
it's ok mate!

in the meanwhile, ill start working on data visualization, at least for new weekly cases. Also, can you document the part of the code where the data gets stored into the txt file - I'll write a wrapper function to feed it into the datahandle.py

**Comment by renz-b** [comment]
Hello! Sorry I couldnt get back to you sooner. Work load has been piling up and havent even done coding for the past week. Hope you understand. Im sorry

**Comment by grokwithrahul** [comment]
it's ok :)

**Comment by renz-b** [comment]
Good morning!

This is the function that writes the data to a txt file found in _**image_to_txt.py**_

    def write_to_file(data):
        os.chdir(root)
        with open('data.txt', 'a') as file:
            file.write('{}\n'.format(str(data)))


It is referenced twice in **_main.py_**. In line 39 inside the `edit_data_before_tweet` function and in line 91 in `main`.

I have a short free time today and will try to merge the tiny db pull request:)

**Comment by renz-b** [comment]
Hello!

Since I wanted to keep the master branch clean before adding PRs and testing functionality I added a new branch called test!

I used git fetch and pulled your code to the branch and did some changed to fit the existing code. Its working and adds to the .json files in the data folder. :) you can check it out I just pushed not too long ago.


This is just my understanding to keep the main code clean. If I did something wrong please tell me and that I am unaware of common and correct coding practices.

Thank you!

**Comment by grokwithrahul** [comment]
great :)
